SharedResources/en.lproj/Localizable.strings

```diff
@@ -90,6 +90,16 @@
 "fileProvider.onboarding.info" = "Thanks for choosing Cryptomator to protect your files. To get started, go to the main app and add a vault.";
 "fileProvider.onboarding.button.openCryptomator" = "Open Cryptomator";
 
+"keepUnlocked.alert.title" = "Lock Vault?";
+"keepUnlocked.alert.message" = "This change requires your vault to be locked in order to take effect after your next unlock.";
+"keepUnlocked.alert.confirm" = "Confirm & Lock Now";
+"keepUnlocked.header" = "Specify how long you want this vault to be unlocked after last activity.";
+"keepUnlocked.footer.auto" = "The selected option means that Cryptomator may be terminated in the background by the Files app to free up memory, which automatically locks the vault.";
```

Comment:
```suggestion
"keepUnlocked.footer.auto" = "Letting iOS decide means that Cryptomator may be terminated any time to free up memory, which automatically locks the vault.";
```

1. Instead of generic "selected options" we make an explicit reference to _what_ option is selected
2. does "in the background" add any relevant information?